frontend/src/app/general/confirmation-dialog/confirmation-dialog.component.html

```diff
@@ -3,7 +3,7 @@
  ~ SPDX-License-Identifier: Apache-2.0
```

Comment:
The component belongs to the `helpers` directory, not the `general` directory. General contains some core features while helpers contains reusable components.

Reply:
I made a separate commit for this as it does not directly belong to this PR but easier to include it here now as I already applied two minor improvements to the confirmation dialog.
In addition, I already moved the input dialog to the helper directory. This was not mentioned by you but here the same applies as for the confirmation dialog.